docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -142,40 +147,42 @@
     crop_max_scales=[1.0, 1.0],
 )
 
-# Single crop augmentation for MAE
+# Single crop augmentation for MAE
 mae_collate_fn = lightly.data.MAECollateFunction()
 
 # Multi crop augmentation for MSN
 msn_collate_fn = lightly.data.MSNCollateFunction(random_size=128, focal_size=64)
 
+# Collate function passing geometrical transformation for VICRegL
+vicregl_collate_fn = lightly.data.VICRegLCollateFunction()
```

Comment:
Oh no, we usually use smaller image sizes than default for the imagenette benchmarks as the images are only 160px large. The global crop size is controlled by `input_size` variable set at the top of the file. And I would set local size to 64.